package.json

```diff
@@ -44,6 +44,7 @@
     "angular-ui-bootstrap": "^2.5.0",
     "angular-ui-sortable": "^0.17.0",
     "angular2react": "^3.0.2",
+    "ansi_up": "^4.0.4",
```

Comment:
I don't have a strong opinion, but just curious why you chose this library over the [seemingly more popular alternative](https://www.npmjs.com/package/ansi-to-html) (1.6 million weekly downloads vs. 18k)

Reply:
No reason actually, weird that this library didn't show up when I searched. I'll try it out

Reply:
I think I'll stay with ansi_up:
* It has double the number of stars on github (which I think is a better popularity measures than number of downloads).
* The other library does not have type support. In fact, type support was introduced 4 months ago, but the author forgot to upload the new version to npm. Not a great sign :sweat_smile:

Reply:
sounds good, thanks for checking it out!